### ISSUE TITLE
TEE-verified badge

### ISSUE THREAD

**Issue by pbeza** [matter-labs/block-explorer#325]
#### 📝 Description

We [are now generating TEE proofs](https://x.com/portport255/status/1834494086981558768) alongside ZK proofs as part of the [multi-prover work](https://ethresear.ch/t/2fa-zk-rollups-using-sgx/14462). It would be great to show TEE proofs in the block explorer, along with info on how to verify them.

Once we have [on-chain verification](https://github.com/automata-network/automata-dcap-attestation) in place, we can add a badge next to each batch that automatically shows the result of the on-chain TEE proof verification. For now, the best way to verify TEE proofs is by using the Docker image [we provide](https://x.com/portport255/status/1834498408260010187).

#### 🤔 Rationale

Users should be able to easily check the TEE proofs we make if they want to.

#### 📋 Additional Context

- Demo: https://x.com/portport255/status/1834494086981558768
- Getting the TEE proof for sample batch number `494384`:
   ```
   curl -X POST \
     -H "Content-Type: application/json" \
     --data '{"jsonrpc": "2.0", "id": 1, "method": "unstable_getTeeProofs", "params": [494384, "sgx"] }' \
     https://mainnet.era.zksync.io
   ```

**Comment by popzxc** [comment]
@pbeza Probably this part is to be handled by another issue, since it's not actionable now.

> Once we have [on-chain verification](https://github.com/automata-network/automata-dcap-attestation) in place, we can add a badge next to each batch that automatically shows the result of the on-chain TEE proof verification. For now, the best way to verify TEE proofs is by using the Docker image [we provide](https://x.com/portport255/status/1834498408260010187).


Maybe it would be fair to display the badge together with the link to the proof now, even though it's not enforced by L1 contract? We can add hover text on the badge saying that the proof can be verified by anyone, but is currently not enforced by the L1 contract.

**Comment by pbeza** [comment]
> Probably this part is to be handled by another issue, since it's not actionable now.

Yep, that was my intention – to leave it for now since we haven’t integrated TEE proof on-chain verification yet.

> Maybe it would be fair to display the badge together with the link to the proof now, even though it's not enforced by L1 contract? We can add hover text on the badge saying that the proof can be verified by anyone, but is currently not enforced by the L1 contract.

Sounds fine to me – I don’t have a strong opinion on this. cc-ing @haraldh in case he has a better idea.

**Comment by haraldh** [comment]
Yeah, a `Verified by TEE` badge which links to a page explaining what it means and how to verify manually.
